apps/builder/app/builder/features/style-panel/sections/flex-child/flex-child.tsx

```diff
@@ -108,6 +108,7 @@ const FlexChildSectionSizing = (props: RenderCategoryProps) => {
       <PropertyName
         style={currentStyle}
         properties={["flexGrow", "flexShrink"]}
+        shorthandProperty="sizing"
```

Comment:
lets find a better name for shorthandProperty since sizing is not a real css shorthand, mb:
"group" or "category"?